hack/s3_test/test.sh

```diff
@@ -20,7 +20,17 @@ done
 
 export default_options="type=s3,bucket=my-bucket,region=us-east-1,endpoint_url=http://127.0.0.1:9000,access_key_id=minioadmin,secret_access_key=minioadmin,use_path_style=true"
 
-rm -rf /tmp/destdir1 /tmp/destdir2
+rm -rf /tmp/destdir0 /tmp/destdir1 /tmp/destdir2
+
+# Check --export-cache with ignore-error=true and fake non reachable endpoint_url
+# Should ignore error and skip the remote cache export
```

Comment:
Putting the test in here isn't ideal. For example, look `testZstdLocalCacheExport` as an example for a real integration test. Add both the failure case with `ignore-error=false` and the case that skips the error.

Reply:
Thanks for the example, I didn't find out firstly such tests, this is well better now indeed :ok_hand:

I've added a use case for cache-export type: local, registry and s3
For cache-export type: gha and azblob, it's more difficult to test with a fake url so I just didn't add them